### PR TITLE
fix(console): disable verbose opentelemetry if it is disabled

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-7516

## Description

A small description of what you did in that PR.

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-kubouzuiib.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.ts [hunk 1]
```diff
@@ -221,6 +222,20 @@ export class ApiRuntimeLogsMessageSettingsComponent implements OnInit, OnDestroy
...
+  private handleTracingEnabledChanges(): void {
+    this.form
+      .get('tracingEnabled')
+      .valueChanges.pipe(takeUntil(this.unsubscribe$))
+      .subscribe((tracingEnabled) => {
+        if (!tracingEnabled) {
+          this.disableAndUncheck('tracingVerbose');
+        } else {
+          this.form.get('tracingVerbose').enable();
+          this.form.get('tracingVerbose').setValue(false);
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.ts]: How about leaving it as is and just disabling the button? 
The idea is just to make the user understand that this button is not deactivated now. But not to change its configuration/value.
it's classic behavior. already present on other settings screen

Comment (illichso) [reply gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.ts]: Good point. Let me update

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-proxy-settings/api-runtime-logs-proxy-settings.component.ts [hunk 2]
```diff
@@ -177,8 +184,8 @@ export class ApiRuntimeLogsProxySettingsComponent implements OnInit {
             this.form.get('headers').disable();
             this.form.get('payload').disable();
             this.form.get('condition').disable();
+            this.form.get('tracingEnabled').setValue(false);
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-proxy-settings/api-runtime-logs-proxy-settings.component.ts]: is not necessary with last change no  ?

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-proxy-settings/api-runtime-logs-proxy-settings.component.ts (hunk 2 above)
Comment (illichso) [reply gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-proxy-settings/api-runtime-logs-proxy-settings.component.ts]: Removed

**Pull request merged**